src/storage/migration.rs

```diff
@@ -309,6 +309,10 @@ fn migrate_0_24_to_0_26(db_path: &Path, db_opts: &DbOptions) -> Result<()> {
     let db = Database::open(db_path, db_opts)?;
     migrate_0_24_to_0_26_conn(&db)?;
     migrate_0_24_to_0_26_http(&db)?;
+    migrate_0_24_to_0_26_smtp(&db)?;
```

Comment:
Other protocols generated by Piglet, including Rdp, Dhcp, and DNS, should also be included in this fix.